content/_announcements/jan2020-new-website.md

```diff
@@ -4,25 +4,37 @@ title: Our new website
 announcement-date: January 20, 2020
```

Comment:
Should we change the announcement date to whichever day we'll announce?

Reply:
I'll set it a day or two ahead when I push the final version.